content/2020/10-26-new-folder-lock-apis-now-available.md

```diff
@@ -0,0 +1,47 @@
+---
+applied_at: "2020-10-26"
+applies_to: 
+- api
+is_impactful: false
+is_new_feature: true
+collapse: true
+show_excerpt: true
+release_source_url: ''
+---
+
+# New Folder Lock APIs now available
+
+A new collections of APIs have been released to allow developers to lock move
```

Comment:
```suggestion
A new collections of APIs have been released to prevent developers to lock move
```
This might be a better way to describe what locking is without using the word lock?

Reply:
Adjusted it to flow a little differently: A new collections of APIs have been released to permit the restriction of move
and delete actions on folders